spec/spec_helper.rb

```diff
@@ -8,6 +8,7 @@
 
 Combustion.initialize! :all do
   config.cache_store = :null_store
+  config.action_dispatch.show_exceptions = :all
```

Comment:
```suggestion
  config.action_dispatch.show_exceptions = true
```

Do we need :all here? I imagine a boolean would be easier for someone to follow. It'd also be good to have either the commit message or a comment document why we've had to turn this on

Reply:
the true has been deprecated and it now recommends `:all`

Reply:
Ah super.